web/app/scripts/black-spots/black-spots-controller.js

```diff
@@ -1,11 +1,73 @@
-(function () {
+(function() {
     'use strict';
 
     /* ngInject */
-    function BlackSpotsController() {
+    function BlackSpotsController(
+        InitialState, TileUrlService, FilterState, RecordState, BlackspotSets
+    ) {
+        var cartoDBAttribution = '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, &copy; <a href="http://cartodb.com/attributions">CartoDB</a>';
+        var ctl = this;
+        ctl.map = null;
+        ctl.initMap = initMap;
+        initialize();
```

Comment:
If there's no code being run on initialize, this call and the function can be removed.